funnel/models/project.py

```diff
@@ -382,6 +382,12 @@ def datelocation(self):
         lambda project: db.func.utcnow() < project.schedule_start_at,
         label=('upcoming', __("Upcoming")),
     )
+    schedule_state.add_conditional_state(
+        'PUBLISHED_WITHOUT_SESSIONS',
+        schedule_state.PUBLISHED,
+        lambda project: project.schedule_start_at is None,
```

Comment:
This is the Python-side query. You need a second lambda for the SQL-side query. Add this line immediately after the first lambda:
`lambda project: project.schedule_start_at.is_(None),`